### ISSUE TITLE
Gridcoin payment add-on for Wordpress

### ISSUE THREAD

**Issue by Erkan-Yilmaz** [Erkan-Yilmaz/Gridcoin-tasks#118]
Who wants to help ?

* gciapita: is anyone working on a gridcoin payment add-on for wordpress?

* Erkan_Yilmaz: it exists already, !coinpayments https://wordpress.org/plugins/coinpayments-payment-gateway-for-woocommerce/
* fediverse: You can buy products with Gridcoin in these shops: https://github.com/Erkan-Yilmaz/Gridcoin-tasks/issues/28
* gciapita: it doesnt erkan
*gciapita: coinpayments is full of dodgy businesses and i want no affiliation with drug dealers and pornographers, etc.
* Erkan_Yilmaz: well then you have to code one, I guess
* gciapita: i know

* BullShark: Most shop owners would want it to convert to FIAT on the back-end

**Comment by denravonska** [comment]
Doesn't the Coinpayments plugin cover this? https://www.coinpayments.net/merchant-tools-plugins

**Comment by grctest** [comment]
Yes, coinpayments does cover this but gciapita doesn't want to associate himself with coinpayments because allegedly dodgey websites use it.. this is the case for all businesses, not just coinpayments, so i think it should be fine just using coinpayments..

**Comment by MagixInTheAir** [comment]
Any update ? Is the developpement still going ?
